code/data-tracker/data-tracker.js

```diff
@@ -11,12 +11,18 @@ function customizeLoginButton(viewId) {
 
   // Create a div for Login buttons
   var $coacdButton = $("<div/>", {
-    id: "coacd-button-login"
+    id: "coacd-button-login",
   });
   $coacdButton.appendTo("#" + viewId);
 
   // Append Big SSO Login button and non-SSO Login button
-  bigButton("coacd-big-button", "coacd-button-login", url, "sign-in", "Sign-In")
+  bigButton(
```

Comment:
Curious on the reasoning of re-expanding the SSO code to take up more lines?

Reply:
this is the default formatting behavior of [Prettier](https://prettier.io/). i actually was surprised it picked up more linting, since i linted this code the last time i edited it a few months ago. it makes me wonder if knack does some kind of linting automatically, or if my linter settings changed somehow.

i'm happy to defer to the linter's opinions, which aim for more readable code over being concise. in the past we've talked about standardizing these settings and making sure we use them globally, basically the standard right now it prettier's default.